static/js/apps/eval_embeddings/app.tsx

```diff
@@ -155,37 +160,28 @@ const fetchEmbeddings = async (lines: string[], modelNames: string[]) => {
     }
```

Comment:
To confirm, is this tolerant of additional commas in desc1, desc2?

`dcid1,desc1 with, comma;desc2 with, comma`

Reply:
very good point. use a real csv parser now